HIPs/hedera-hashgraph.md

```diff
@@ -0,0 +1,313 @@
+---
+layout: default
+title: PROJECT
+parent: Hyperledger Improvement Proposals
+---
+
+# Sponsor(s)
+- Alex Popowycz (Hedera) [<a@hedera.com>](mailto:a@hedera.com)
+- Samson Goddy (Hedera) [<samson@hedera.com>](mailto:samson@hedera.com)
+- Richard Bair (Hashgraph) [<richard@hashgraph.com>](mailto:richard@hashgraph.com)
+- Leemon Baird (Independent) [<leemon@swirldslabs.com>](mailto:leemon@swirldslabs.com)
+- Hendrik Ebbers (Hashgraph) [<hendrik.ebbers@hashgraph.com>](mailto:hendrik.ebbers@hashgraph.com)
+- May Chan (HashPack) [<may@hashpack.app>](mailto:may@hashpack.app)
+- Shyam Nagarajan (IBM) [<shyam@us.ibm.com>](mailto:shyam@us.ibm.com)
+- Suma Nair (IBM) [<sumapnair@us.ibm.com>](mailto:sumapnair@us.ibm.com)
+- Katerina Sanchez-Schilling (DLT Science Foundation) [<Katerina.Sanchez-Schilling@dltscience.org>](mailto:Katerina.Sanchez-Schilling@dltscience.org)
+- Paolo Tasca (DLT Science Foundation) [<pt@dsf.xyz>](mailto:pt@dsf.xyz)
+- Nikhil Vadgama (DLT Science Foundation) [<nv@dsf.xyz>](mailto:nv@dsf.xyz)
+
+# Abstract
+This proposal outlines the plan to move Hedera’s core network software to the Linux Foundation’s Decentralized Trust Foundation.
+The Hedera network is a public distributed ledger technology (DLT) offering high throughput, low latency, and strong security.
+We are requesting to contribute Hedera’s core network software which includes our unique hashgraph consensus algorithm
+and related software and governance model to the Linux Fouundation's Decentralized Trust ecosystem.
+We want to leverage and contribute to the existing community and resources for further development and adoption across
+the Decentralized Trust ecosystem in a vendor-neutral process.
+Under the new foundation, the project will be rebranded and trademarked as `PROJECT`. 
+
+# Context
+`PROJECT` offers a robust decentralized ledger platform leveraging a unique consensus mechanism known as Hashgraph,
+which provides asynchronous Byzantine Fault Tolerance (aBFT), making it secure, fast, and fair.
+By joining with the existing Hyperledger project under the umbrella of LF Decentralized Trust,
+`PROJECT` will aim to enhance collaboration with other blockchain and DLT projects,
+contribute to the open-source community, and accelerate the adoption of enterprise-grade DLT solutions.
+
+# Dependent Projects
+None
```

Comment:
Consensus node and Mirror node use Besu's EVM.  Mirror node uses Web3j.

Reply:
thanks @shemnon, I had the exact same question and now you just answered it.

Reply:
this change was made